include/odp/api/spec/packet_io_types.h

```diff
@@ -265,6 +265,16 @@ typedef struct odp_pktin_queue_param_t {
 	  * Queue type is defined by the input mode. The default value is 1. */
 	uint32_t num_queues;
 
+	/** Input queue size array
+	  *
+	  * An array containing queue sizes for each 'num_queues' input queues.
+	  * The value of zero means implementation specific default size.
+	  * Nonzero values must be between 'min_input_queue_size' and
+	  * 'max_input_queue_size' capabilities. The implementation may
+	  * round-up given values. The default value is zero.
+	  */
+	uint32_t queue_size[ODP_PKTIN_MAX_QUEUES];
```

Comment:
We should mention that this applies to only DIRECT mode.

Reply:
Added mention in V3.